app/views/api_docs/pages/home.md

```diff
@@ -12,13 +12,10 @@ Retrieving information about postgraduate teacher training courses.
 
 ## Codes and reference data
```

Comment:
It's not clear to me what the 'codes' will be used for

Reply:
Same. But I was copying what Apply had and wanted to focus more on other elements, like getting it working. Got a suggestion for better wording?